examples/step-60/step-60.cc

```diff
@@ -0,0 +1,1045 @@
+  // relative path (e.g., ParameterAcceptor("My Other Class")) instead of the
+  // absolute one (e.g. ParameterAcceptor("/first/second/third/My Other Class")),
+  // obtaining:
+  // @code
+  // subsection first
+  //   subsection second
+  //     subsection third
+  //       subsection MyClass
+  //        ... # all the parameters
+  //       end
+  //       subsection My Other Class
+  //        ... # all the parameters of MyOtherClass
+  //       end
+  //     end
+  //   end
+  // end
+  // @endcode
+  //
+  // If the section name *ends* with a slash then subsequent classes will
+  // interpret this as a full path: for example, similar to the one above, if
+  // we have two classes, one initialized with
+  // ParameterAcceptor("/first/second/third/My Class/")
+  // and the other with ParameterAcceptor("My Other Class"), then the
+  // resulting parameter file will look like:
+  //
+  // @code
+  // subsection first
+  //   subsection second
+  //     subsection third
+  //       subsection MyClass
+  //        ... # all the parameters
+  //        # notice MyClass subsection does not end here
+  //        subsection My Other Class
+  //         ... # all the parameters of MyOtherClass
+  //        end
+  //       end # of subsection MyClass
+  //     end
+  //   end
+  // end
+  // @endcode
+  //
+  // We are going to exploit this, by making our
+  // DistributedLagrangeProblemParameters
+  // the *parent* of all subsequently constructed classes. Since most of the other
+  // classes are members of DistributedLagrangeProblem this allows, for example,
+  // to construct two DistributedLagrangeProblem for two different dimensions, without
+  // having conflicts in the parameters for the two problems.
+
+  template<int dim, int spacedim>
+  DistributedLagrangeProblem<dim,spacedim>::DistributedLagrangeProblemParameters::
+  DistributedLagrangeProblemParameters() :
+    ParameterAcceptor("/Distributed Lagrange<" + Utilities::int_to_string(dim)
+                      + "," + Utilities::int_to_string(spacedim) +">/")
+  {
+
+    // The ParameterAcceptor::add_parameter does a few things:
+    //
+    // - enters the subsection specified at construction time to ParameterAcceptor
+    //
+    // - calls the ParameterAcceptor::prm.add_parameter
+    //
+    // - calls any signal you may have attached to
+    // ParameterAcceptor::declare_parameters_call_back
+    //
+    // - leaves the subsection
+    //
+    // In turns, ParameterAcceptor::prm.add_parameter
+    //
+    // - declares an entry in the parameter handler for the given variable;
+    //
+    // - reads the value of the variable,
+    //
+    // - transforms it to a string, used as the default value for the parameter
+    // file
+    //
+    // - attaches an *action* to ParameterAcceptor::prm that monitors when a file
+    // is parsed, or when an entry is set, and when this happens, it updates the
+    // content of the given variable to the value parsed by the string
```

Comment:
I guess this is not going to be a nicely formatted list. As far as I know, we typically use html syntax, i.e., 
```
<ul>
<li> item 1
<li> item 2
</ul>
```
for a bullet list. (Or `<ol> </ol>` for a numbered list.)

Reply:
Actually, this is perfectly well formatted. Doxygen understands markdown, and this is equivalent to doing what you suggested...